modules/lightning_features/lightning_core/modules/lightning_dev/lightning_dev.install

```diff
@@ -54,6 +55,24 @@ function lightning_dev_install() {
   if (\Drupal::moduleHandler()->moduleExists('lightning_api')) {
     lightning_dev_modules_installed(['lightning_api']);
   }
+
+  $image_styles = array_map(
+    function ($crop_type) {
+      return 'crop_' . $crop_type;
+    },
+    \Drupal::entityQuery('crop_type')->execute()
+  );
+  $image_styles = ImageStyle::loadMultiple($image_styles);
+
+  /** @var ImageStyle $image_style */
+  foreach ($image_styles as $image_style) {
```

Comment:
This is necessary because image effects get a new UUID when they're added to an image style. That would cause our exported config to change with every new install.